tethys_cli/start_commands.py

```diff
@@ -0,0 +1,134 @@
+import os
+import webbrowser
+from argparse import Namespace
+from tethys_apps.utilities import (
+    get_installed_tethys_items,
+    get_tethys_home_dir,
+    relative_to_tethys_home,
+)
+from tethys_cli.cli_helpers import get_manage_path, run_process, setup_django
+from tethys_cli.db_commands import configure_tethys_db, process_args
+from tethys_cli.gen_commands import (
+    get_destination_path,
+    generate_command,
+    generate_secret_key,
+    GEN_PORTAL_OPTION,
+)
+from tethys_cli.scaffold_commands import scaffold_command, APP_PREFIX
+from tethys_cli.install_commands import install_command
+from tethys_cli.settings_commands import settings_command
+
+
+def add_start_parser(subparsers):
+    # Setup list command
+    start_parser = subparsers.add_parser(
+        "start",
+        help="Start the tethys server. "
+        "If being done for the first time, this will also generate a portal config file, "
+        "configure the database, and prompt to scaffold an app as well.",
+    )
+    start_parser.add_argument(
+        "-p",
+        "--port",
+        type=str,
+        help="Host and/or port on which to bind the development server.",
+    )
+    start_parser.set_defaults(func=start_command)
```

Comment:
based on the help, it seems this should call `quickstart_command` too. i.e. it’s just an alias for the same function.

Reply:
Good catch. Fixed.